docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -15,6 +15,7 @@ it is supported for production use.
 #### Contents
 
 * [Introduction](#introduction)
+* [Prerequisites for SOA Suite Domains](#prerequisites-for-soa-suite-domains)
```

Comment:
SOA Suite Domains -> SOA Suite domains

Reply:
This is title of the patch : 

BUG 30638100 - SOA BUNDLE PATCH 12.2.1.3.191208(ID:191208.1658.0113)

This is called "Bundle" not "Bundled".

Reply:
I stand corrected. Please ignore those edits.